site/content/api/Device-Registration-API.md

```diff
@@ -58,15 +58,17 @@ A response to an *assertion* request contains the [Standard Response Properties]
 
 The body of the response message consists of a single *Data* section containing a UTF-8 encoded string representation of a single JSON object having the following properties:
 
-| Name             | Mandatory | JSON Type     | Description |
-| :--------------- | :-------: | :------------ | :---------- |
-| *device-id*      | *yes*     | *string*      | The ID of the device that is subject of the assertion. |
-| *defaults*       | *no*      | *object*      | Default values to be used by protocol adapters for augmenting messages from devices with missing information like a *content type*. It is up to the discretion of a protocol adapter if and how to use the given default values when processing messages published by the device. |
+| Name             | Mandatory | JSON Type         | Description |
+| :--------------- | :-------: | :---------------- | :---------- |
+| *device-id*      | *yes*     | *string*          | The ID of the device that is subject of the assertion. |
+| *via*            | *yes*     | *string or array* | Information about which gateways may act on behalf of the device. The property may either contain a single gateway device id as a string or a JSON array containing zero or more gateway ids as strings. In case no gateways are involved, an empty JSON array is to be used. |
```

Comment:
why do you need the value itself in the protocol adapter? The authorization decision is made in the Device Registration service. My understanding is that you only need to know, whether the device may connect via a gateway at all, i.e. a boolean would be sufficient, or am I mistaken?

Reply:
Right, the value itself isn't needed. I would use a boolean property then.
As name, I think `via-enabled` would do (`access-via-gateway-enabled` or `via-gateway-usage-enabled` being a bit long).

Reply:
what about `gw-supported`?

Reply:
yes, good. I've changed it to that.